specification/devopsinfrastructure/Microsoft.DevOpsInfrastructure/main.tsp

```diff
@@ -23,6 +23,7 @@ namespace Microsoft.DevOpsInfrastructure;
 enum Versions {
```

Comment:
Please if you can, split the files into smaller manageable files (maybe by resources)?

Reply:
Will do as a separate PR, want to get this change out as soon as possible.